desktop/containerd/index.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Containerd Image Store (Beta)
+description: How to activate the containerd integration feature in Docker Desktop
+keywords: Docker, containerd, engine, image store, lazy-pull
+toc_max: 3
+---
+
+This page provides information about the ongoing integration of `containerd` for image and file system management in the Docker Engine.
+
+> **Beta**
+>
+> The Containerd Image Store feature is currently in [Beta](../../release-lifecycle.md/#beta). We recommend that you do not use this feature in production environments as this feature may change or be removed from future releases.
+
+
+## Enabling the Containerd image store feature
+
+The Containerd Image Store beta feature is switched off by default.
+
+To enable this feature:
+1. Navigate to **Settings**, or **Preferences** if you’re a Mac user.
+2. Select the **Experimental** features tab.
+3. Next to **Use containerd for pulling and storing images**, select the checkbox.
+
+To disable this feature uncheck **Use containerd for pulling and storing images**.
```

Comment:
```suggestion
To disable this feature, clear the **Use containerd for pulling and storing images** checkbox.
```

Reply:
https://docs.gitlab.com/ee/development/documentation/styleguide/word_list.html#checkbox
Also what we follow in our style guide. 
Super picky point, the rest looks great

Reply:
Just to clarify, you sent a GitLab docs style link? I'm committing your review suggestion. Let me know when you have a go for Desktop docs/release notes so that I can publish this one as well. Thanks!

Reply:
Will let you know!